test/gherkin_helpers.ts

```diff
@@ -15,6 +16,9 @@ export interface IParsedSourceWithEnvelopes extends IParsedSource {
 export async function parse({
   data,
   uri,
+}: {
+  data: string
+  uri: string
```

Comment:
Can we create an `IParseRequest`? Same for below

Reply:
Done and done